recipes/bolt/bld.bat

```diff
@@ -0,0 +1,43 @@
+@echo on
+
+:: until https://github.com/llvm/llvm-project/pull/97130 lands,
+:: follow https://github.com/conda-forge/llvmdev-feedstock/blob/main/recipe/build.sh,
+:: minus the tests and plus LLVM_ENABLE_PROJECTS="bolt" / LLVM_TARGETS_TO_BUILD=...
+
+mkdir build
+cd build
+
+REM remove GL flag for now
+set "CXXFLAGS=-MD"
+set "CC=cl.exe"
+set "CXX=cl.exe"
+
+cmake -G "Ninja" ^
+    -DCMAKE_BUILD_TYPE="Release" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreadedDLL ^
+    -DLLVM_USE_INTEL_JITEVENTS=ON ^
+    -DLLVM_ENABLE_DUMP=ON ^
+    -DLLVM_ENABLE_LIBXML2=FORCE_ON ^
+    -DLLVM_ENABLE_PROJECTS="bolt" ^
+    -DLLVM_ENABLE_RTTI=ON ^
+    -DLLVM_ENABLE_ZLIB=FORCE_ON ^
+    -DLLVM_ENABLE_ZSTD=FORCE_ON ^
+    -DLLVM_INCLUDE_BENCHMARKS=OFF ^
+    -DLLVM_INCLUDE_DOCS=OFF ^
+    -DLLVM_INCLUDE_EXAMPLES=OFF ^
+    -DLLVM_INCLUDE_TESTS=ON ^
+    -DLLVM_INCLUDE_UTILS=ON ^
+    -DLLVM_INSTALL_UTILS=ON ^
+    -DLLVM_TARGETS_TO_BUILD="X86;AArch64" ^
+    -DLLVM_USE_SYMLINKS=OFF ^
+    -DLLVM_UTILS_INSTALL_DIR=libexec\llvm ^
+    -DLLVM_BUILD_LLVM_C_DYLIB=ON ^
+    -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly ^
```

Comment:
Remove

Reply:
This is all from the llvmdev-recipe. If you want me to stop mirroring that (which was the in https://github.com/conda-forge/llvmdev-feedstock/pull/276, with @xhochy voicing concerns about divergence), please say so.

Reply:
Yes, I said we should remove all targets except x86_64 and aarch64 at https://github.com/conda-forge/llvmdev-feedstock/pull/276#issuecomment-2245814101

Reply:
It wasn't clear that you're just talking about a single line here (the previous comment was about a block). In any case, done in https://github.com/conda-forge/staged-recipes/pull/22204/commits/004cdc9409acdfe614a1c78582208c1ab564b1a4